assets/css/pets-love.css

```diff
@@ -0,0 +1,48 @@
+.pets-love-wrapper{
+    height:70vh;
+    width:100%;
+    display:flex;
+}
+.pets-love-heading{
+    width:33.33%;
+    height:80%;
+    margin:auto 0;
+    padding:30px 30px 30px 30px;
+}
+.pets-love-heading h4{
+    margin-bottom:20px;
+    padding-top:250px;
+    text-transform: uppercase;
+    letter-spacing: 4px;
+}
+.pets-love-photo{
+    overflow:hidden;
+    height:80%;
+    width:33.34%;
+    margin:auto 0;
+}
+img{
+    top:50%;
+    transform:translateY(0%);
```

Comment:
To nic nie daje. Może chciałeś dać tutaj -50? Aktualnie kazałeś elementowi przesunąć się o 0% względem swojego aktualnego położenia.